components/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+const api = axios.create({
+  baseURL: "https://chriss-new-app.onrender.com",
+});
+
+
+
+export const Home = () => {
+    const [articles, setArticles] = useState([])
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() =>{
+        api.get('/api/articles')
+        .then((response) => {
+            console.log(response.data.articles)
+            setArticles(response.data.articles)
+            setLoading(false);
+        })
+    }, [])
+    if(loading) {
+        return <div>Loading!</div>
+    }
+
+    return <div className="home">
+        <h2>Articles</h2>
+        <h3>Sort By Topics/Authors</h3>
+        
+        <ul>
+            {articles.map((article) => {
```

Comment:
Please make sure you have no errors in the browser console before submitting - you need a key prop for each of the elements you are mapping over here.

Speaking of the console - line 18 👀 Please remove before committing in future